devel-requirements.txt

```diff
@@ -0,0 +1,10 @@
+-r requirements.txt
+
+responses
+pytest
+pytest-cov
+mock
+
+pylint
+flake8
+pep8
```

Comment:
I prefer requirements with `==`

If in the future any of these libraries push a change which is not back-compatible our project might be affected.